### ISSUE TITLE
Screen reader doesn't read contact form input labels

### ISSUE THREAD

**Issue by bobbysebolao** [FAC-Sixteen/BBFS#20]
Chromevox screen reader doesn't read the contact form labels when you tab through the content on your keyboard. I think this is because the `<input>` elements do not have an `id` that matches the `for` attribute on their label. To fix, just add a matching `id` to each input 😄

**Comment by charlielafosse** [comment]
Good spot - relates to my comment in #12

**Comment by sylv1ah** [comment]
thanks for the feedback, we resolved it in a previous issue! 😄
